test/extensions/INTEL/SPV_INTEL_joint_matrix/joint_matrix_checked.ll

```diff
@@ -0,0 +1,157 @@
+; This is an adapted copy of test/extensions/INTEL/SPV_INTEL_joint_matrix/cooperative_matrix_checked.ll
+
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_joint_matrix -o %t.spv
+; RUN: llvm-spirv %t.spv -to-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-SPIRV: Capability JointMatrixINTEL
```

Comment:
Spec says that we should also enable `CooperativeMatrixCheckedInstructionsINTEL` capability for checked instructions. Can you please add check for that?

Reply:
thank you, fixed